content/docs/user-guide/data-management/discovering-and-accessing-data.md

```diff
@@ -1,10 +1,4 @@
----
-title: 'Get Started: Data and Model Access'
-description: 'Get started with accessing data and models with DVC. Learn how to
-bring, explore, and access data artifacts from outside the project'
----
-
-# Get Started: Data and Model Access
+# Discovering and accessing data
```

Comment:
some text on this page becomes out of context I think

Reply:
gave this another read, lgtm. Did some minor copy-editing to the initial paragraph.
To clarify, I don't think referring to the 'get-started' dir of https://github.com/iterative/dataset-registry is problematic.